sdf.md

```diff
@@ -738,7 +738,14 @@ referencing SDF document they build the SDF model expressed by that
 SDF document.
 
 Each SDF document is represented as a single JSON map.
-This map has three blocks: the information block, the namespaces block, and the definitions block.
+This map can be thought of as having three blocks: the information
+block, the namespaces block, and the definitions block.
+These blocks contain zero or more JSON name/value pairs, the names of
+which are quality names and the values of which mostly are (nested)
+maps (the exception defined in SDF base is the defaultNamespace
+quality, the value of which is a text string).
+An empty nested map of this kind is equivalent to the absence of the
+name/value pair it is the value of.
```

Comment:
I was wondering if there's a more clear way to say this, but the best I came up with was:
> An empty nested map of this kind is equivalent to not having the quality included at all

Reply:
I also stumbled upon that sentence, I think your suggestion sounds good, @akeranen :)